ui/src/VoxettaApp.js

```diff
@@ -60,28 +102,102 @@ export class VoxettaApp extends LitElement {
      */
     displayComponents() {
         switch (this.state) {
```

Comment:
nit: This could be the render function since the render function calls this function and does nothing else.

Reply:
moved body of displayComponents() to render